extensions/amp-stream-gallery/1.0/stream-gallery.js

```diff
@@ -99,7 +100,10 @@ export function StreamGallery(props) {
       return;
     }
     // Use local window.
-    const win = node.ownerDocument.defaultView;
+    const win = toWin(node.ownerDocument.defaultView);
```

Comment:
Now come to think of this, if we check `if (!win)`, we actually don't need `toWin` - that's all it does; removes the nullable from type union.

Reply:
I created a `getWin` which can return `?Window` since both effects use it. Instead of removing the `if (!win)`, I removed use of `toWin`. WDYT?